code/src/java/pcgen/persistence/lst/StatsAndChecksLoader.java

```diff
@@ -101,7 +101,7 @@ else if ("ALIGNMENTNAME".equals(key))
 			return null;
 		}
 		String name = token.substring(colonLoc + 1);
-		if ((name == null) || (name.isEmpty()))
+		if (name.isEmpty())
```

Comment:
in this case name can't possibly be null since token.substring never returns null. I don't think this will cause issues.